src/SprykerEco/Client/Computop/Zed/ComputopStub.php

```diff
@@ -70,6 +70,19 @@ public function saveIdealInitResponse(QuoteTransfer $quoteTransfer)
         return $quoteTransfer;
     }
 
+    /**
```

Comment:
* Add the `@uses` document block that points to the gateway controller.
* RFC - https://spryker.atlassian.net/wiki/spaces/RFC/pages/991985877.

Reply:
Done